pil-analyzer/src/type_builtins.rs

```diff
@@ -85,6 +89,11 @@ lazy_static! {
     .into_iter()
     .map(|(op, (vars, ty))| (op, parse_type_scheme(vars, ty)))
     .collect();
+    static ref CONSTR_FUNCTION_STATEMENT_TYPE: ExpectedType = ExpectedType {
```

Comment:
```suggestion

    static ref CONSTR_FUNCTION_STATEMENT_TYPE: ExpectedType = ExpectedType {
```

Reply:
Had to rollback this change to avoid an error with `cargo fmt`